README.md

```diff
@@ -224,6 +262,43 @@ first and last commits that definition has been seen in, and when those commits
 }
 ```
 
+### Glean dependencies files
+
+The Glean dependency file contains information about the dependencies of an
+application in `repositories.yaml`.
+
+The format is similar for Glean metrics data files. The only data point tracked
+in the `history` log is `version`, which is the version number of the
+dependency. At the top-level of each entry is:
+
+- `name`: the name of the dependency (for Android, this is a Maven
+  package name).
+- `type`: Always `"dependency"`.
+
+```
+{
+  "<library name>": {
```

Comment:
Should this be `<application name>`?

Reply:
Each entry here is a library that the application depends on.  I could probably make that clearer in the docs.